front/src/pages/CTF/CTF.vue

```diff
@@ -121,6 +121,7 @@ export default {
 }
 .blur:hover {
   filter: blur(0px);
+  transition-delay:0.2s;
```

Comment:
Please add a space between the property and its value.

Reply:
@XeR Done!